### ISSUE TITLE
Android Statusbar is not tranparent in 0.63.4 version

### ISSUE THREAD

**Issue by vinaygoyal20** [facebook/react-native#31571]
In 0.63.3 the same code  is working fine bug in 0.63.4 statusbar is always white 

## React Native version:0.63.4
![img](https://user-images.githubusercontent.com/26671420/119225736-3efdc500-bb23-11eb-8a29-4bc40a8a7fd1.jpeg)

##Android Version is 10

**Comment by safaiyeh** [comment]
@vinaygoyal20 thanks for the issue! Can you provide a repro that demonstrates this? Is it just the generated template that shows this?

Thanks!

**Comment by annieneedscoffee** [comment]
Are you using react-native-elements or some other dependency that might be overwriting your attempts to change the status bar? This is what happened to me. After updating from "react-native-elements": "^2.0.4" to "react-native-elements": "^3.4.2" I experienced this. I was able to fix it by setting the props like so inside of the react-native-elements header I was using: 

statusBarProps={{ backgroundColor: 'rgb(17, 0, 233)' }}
